lib/src/types/type_registry.dart

```diff
@@ -19,27 +18,40 @@ class TypeOid {
   static const bigIntegerArray = 1016;
   static const boolean = 16;
   static const booleanArray = 1000;
+  static const box = 603;
   static const byteArray = 17;
   static const character = 1042;
+  static const circle = 718;
   static const date = 1082;
+  static const daterange = 3912;
   static const double = 701;
   static const doubleArray = 1022;
+  static const int4range = 3904;
+  static const int8range = 3926;
   static const integer = 23;
   static const integerArray = 1007;
   static const interval = 1186;
   static const json = 114;
   static const jsonb = 3802;
   static const jsonbArray = 3807;
+  static const line = 628;
+  static const lseg = 601;
   static const name = 19;
   static const numeric = 1700;
+  static const numrange = 3906;
+  static const path = 602;
   static const point = 600;
+  static const polygon = 604;
   static const real = 700;
   static const regtype = 2206;
   static const smallInteger = 21;
   static const text = 25;
   static const textArray = 1009;
+  static const time = 1083;
   static const timestampWithoutTimezone = 1114;
   static const timestampWithTimezone = 1184;
+  static const tsrange = 3908;
+  static const tstzrange = 3910;
```

Comment:
This could be `timestampTzRange` instead of `timestampWithTimezoneRange` (also in `types.dart`). We should alias/migrate the `timestampWithTimezone` to `timestampTz` independently to this PR.